frontend/nuxt.config.ts

```diff
@@ -63,6 +63,10 @@ export default defineNuxtConfig({
     },
   },
 
+  nitro: {
```

Comment:
> Before providing introductions on how to enable Brotli for your Nuxt.js application, please be aware that it would be better to let either your [reverse proxy/server](https://www.brotli.pro/enable-brotli/servers/) (like [NGINX](https://www.brotli.pro/enable-brotli/servers/nginx/) or [Apache](https://www.brotli.pro/enable-brotli/servers/apache/)) or your platform (e.g. [Cloudflare](https://www.brotli.pro/enable-brotli/platforms/cloudflare/)) handle Brotli for you. Only if not possible otherwise, you should resort to the instructions.

Reply:
![image](https://github.com/user-attachments/assets/62e85403-d31a-431a-8bf1-eb467146be6a)